docs/dev/run_the_etl.rst

```diff
@@ -11,24 +11,182 @@ just want to use already processed data.
+the job. The bottom part of the window contains dagster logs. You can
+view logs from the ``pudl`` package in the CLI window the dagit process
+is running in.
+
+Once the raw FERC databases are created by a ``pudl.ferc_to_sqlite`` job,
+you can execute the main PUDL ETL.
+
+.. note::
+
+  Make sure you've extracted the raw FERC years you are planning to process
+  with the main PUDL ETL. Jobs in the ``pudl.etl`` definition will fail if
+  the raw FERC database are missing requested years. For example, if you want
+  to process all years available in the ``pudl.etl`` definition make sure
+  you've extracted all years of the raw FERC data.
+
+Select one of the ``pudl.etl`` jobs.
+This will bring you to a window that displays all of the asset dependencies
+in the ``pudl.etl`` definition. Subsets of the ``pudl.etl`` asset graph
+are organized by asset groups. These groups are helfpul for visualizing and
+executing subsets of the asset graph.
+
+To execute the job, select ``fast_etl`` or ``full_etl`` and click "Materialize all".
+You can congifure which years to process by shift+clicking "Materialize all".
+To view the status of the run, click the date next to "Latest run:".
+
+.. image:: ../images/dagit_pudl_etl.png
+  :width: 800
+  :alt: Dagit pudl_etl
+
+You can also re-execute specific assets by selecting one or
+multiple assets in the "Overview" tab and clicking "Materialize selected".
+This is helpful if you are updating the logic of a specific asset and don't
+want to rerun the entire ETL.
+
+.. note::
+
+  Dagster will throw an ``DagsterInvalidSubsetError`` if you try to
+  re-execute a subset of assets produced by a single function. This can
+  be resolved by re-materializing the asset group of the desired asset.
+
+
+Running the ETL with CLI Commands
+---------------------------------
+You can also execute the ETL jobs using CLI commands.
+
+.. note::
+
+  We recommend using Dagit to execute the ETL as it provides additional
+  functionality for re-execution and viewing asset dependences.
+
+There are two main CLI commands for executing the PUDL processing pipeline:
+
+1. ``ferc_to_sqlite`` executes the ``pudl.ferc_to_sqlite`` dagster graph.
+   You must run this script before you can run ``pudl_etl``.
+2. ``pudl_etl`` executes the ``pudl.etl`` asset graph.
+
 Settings Files
 --------------
-These scripts use YAML settings files in place of command line arguments. This avoids
-undue complexity and preserves a record of how the script was run. The YAML file
-dictates which datasets, years, tables, or states get run through the the processing
+These CLI commands use YAML settings files in place of command line arguments.
+This avoids undue complexity and preserves a record of how the script was run.
+The YAML file dictates which years, or states get run through the the processing
 pipeline. Two example files are deployed in the ``settings`` folder that is created when
 you run ``pudl_setup``. (see: :ref:`install-workspace`).
```

Comment:
How should we deal with distributing / deploying the settings files once we get rid of the `pudl_setup` script?

Reply:
Similarly, are these settings files used in the dagster run of the etl too? Right now it seems like they are just related to the CLI etl, but I feel like that's not true.

Reply:
@aesharpe yes the `src/pudl/package_data/settings/*.yml` settings files are being used to configure the dagster jobs. Check out the job definitions in `pudl.etl.__init__.py`.

Reply:
@zaneselvans I'm not sure tbh. Would it be ok to just have folks manage their own directory of settings files? Also, we'd like to move people away from the cli commands + settings files to use dagit.

Reply:
I think in the docs it's still unclear when to fiddle with the settings docs. See my top comment from when I submitted my review.

Reply:
We can have people symlink `workspace/settings` dir to `src/pudl/package_data/settings` for easy access, though there's nothing stopping them from just using whatever settings YAMLs they want to create... something like

```
We use YAML files to configure ETL runs. The repository has two pre-built ones in `src/pudl/package_data/settings`: ``etl_fast.yaml`` and ``etl_full.yaml``.

We use ``etl_fast`` for most local testing as well as testing in CI. It is faster because we only download and ingest a small fraction of the data.

We use ``etl_full`` in our nightly build process because we want to actually ingest all the data.

You are welcome to use these yourself, or create other settings files based on these. The layout of these files is depicted below....
```